homeassistant/components/vicare/climate.py

```diff
@@ -311,8 +311,8 @@ def set_preset_mode(self, preset_mode: str) -> None:
             )
 
         _LOGGER.debug("Current preset %s", self._current_program)
-        if self._current_program and self._current_program != VICARE_PROGRAM_NORMAL:
-            # We can't deactivate "normal"
+        if self._current_program and self._current_program != VICARE_PROGRAM_NORMAL and self._current_program != VICARE_PROGRAM_STANDBY:
+            # We can't deactivate "normal" or "standby"
```

Comment:
Please check if `standby` can also not be activated directly.

Reply:
I think it can't be, I forced it by hardcoding it, and it gave an error, so to be sure I added the same check. At least on my system I have no way of going to standby manually.

Reply:
You find the info in the debug json. There all API data endpoints and their commands are listed.